src/main.go

```diff
@@ -221,3 +227,24 @@ func main() {
 	log.Fatal(http.ListenAndServe(port, nil))
 
 }
+
+func setupLogger() {
+	currTime := time.Now()
+	currFileName := fmt.Sprint(logFilePrefix, currTime.Format("2006-01-02#15:04:05"), ".log")
+	user, err := user.Current()
+	if err != nil {
+		fmt.Printf("cannot access current user data\n")
+		return
+	}
+	homePath := user.HomeDir
+	logFilePath := homePath + "/" + currFileName
+	file, err := os.OpenFile(logFilePath, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
please make sure that when the log file is created or closed, the default access should only be **Read**, i.e., write-protected. this is for security concerns as only the owner or admin retains the right to alert the logs, thereby lessening the confusion at the later stages of the product.

Reply:
Done.